build.gradle

```diff
@@ -113,7 +111,7 @@ ext.tempto_runner = project(':tempto-runner')
 ext.tempto_ldap = project(':tempto-ldap')
 ext.tempto_kafka = project(':tempto-kafka')
 ext.expected_result_generator = project(':expected-result-generator')
-ext.tempto_version = '1.53'
+ext.tempto_version = '1.54'
```

Comment:
This should only be upgraded and set to a non-SNAPSHOT version during the release process. Since we already release 1.53, this should be `1.54-SNAPSHOT`. During release we will update to 1.54, and then move to 1.55-SNAPSHOT afterwards.

So could you set this to 1.54-SNAPSHOT for now?